tests/main/debug-migrate-home/task.yaml

```diff
@@ -0,0 +1,45 @@
+summary: Check the debug command migrate-home migrates snaps' homes
+
+environment:
+    NAME: test-snapd-tools
+
+prepare: |
+  snap pack "$TESTSLIB/snaps/$NAME"
```

Comment:
iirc `install-local` will do the "snap pack" for you so this line should not be needed (some below)

Reply:
Indeed, thanks!